### PR TITLE
Uart debug

### PR DESCRIPTION
**Pull request by dron0gus**

### CODE REVIEW

File: firmware/heater_control.cpp [hunk 1]
```diff
@@ -217,3 +209,23 @@ float GetHeaterDuty()
 {
     return heaterPwm.GetLastDuty();
 }
+
+HeaterState GetHeaterState()
+{
+    return state;
+}
+
+const char* describeHeaterState(HeaterState state) {
```
Comment (mck1117) [inline firmware/heater_control.cpp:218]: format

File: firmware/interpolation.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include "interpolation.h"
+
+float interpolateFloat(float x1, float y1, float x2, float y2, float x)
```
Comment (mck1117) [inline firmware/interpolation.cpp:3]: duplicate of other PR...?

Comment (dron0gus) [reply firmware/interpolation.cpp:3]: This one includes https://github.com/mck1117/wideband/pull/63 . So lets close https://github.com/mck1117/wideband/pull/63 .

File: firmware/heater_control.cpp (hunk 1 above)
Comment (dron0gus) [reply firmware/heater_control.cpp]: Fixed.

**Comment by mck1117** [comment]
this or #59?

**Comment by rusefillc** [comment]
@dron0gus please rebase this one? i assume that we want this to go forward?

**Pull request merged**